code/datums/materials/basemats.dm

```diff
@@ -247,3 +247,15 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	. = ..()
 	if(istype(source, /obj/item))
 		qdel(source.GetComponent(/datum/component/fantasy))
+
+/datum/material/dwarven
+	name = "dwarven alloy"
+	id = "dwarven"
+	desc = "Millenia ago cities of stone and metal were built with this"
```

Comment:
add a period to this. also, it's a terrible description but i'm willing to forgive